tests/integration/resource_manager/test_engine.py

```diff
@@ -42,10 +42,11 @@ def test_create_start_stop_engine(
 ParamValue = namedtuple("ParamValue", "set expected")
 ENGINE_UPDATE_PARAMS = {
     # commented parameters are not available yet
```

Comment:
nit: We can remove this comment now that parameters are available to avoid confusion.